Calmspace/DisplayViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  DisplayViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 17/10/2023.
+//
+
+import UIKit
+import WebKit
+
+class DisplayViewController: UIViewController, WKNavigationDelegate {
+    
+    var link: URL?
+    
+    @IBOutlet weak var webView: WKWebView!
+    
+    @IBOutlet weak var actInt: UIActivityIndicatorView!
```

Comment:
I wouldn't say that `actInt` is a clear variable name. You can of with `activityIndicator`, or, alternatively, `spinner`, or something else that can be easily understood.